platform/appimage/AppRun

```diff
@@ -10,8 +10,9 @@ cd "${KOREADER_DIR}" || exit
 # export load library path
 export LD_LIBRARY_PATH=${KOREADER_DIR}/libs:$LD_LIBRARY_PATH
 
-# export external font directory
-export EXT_FONT_DIR="${HOME}/.fonts"
+# create user font directory if it does not exist.
+USER_FONT_DIR="${HOME}/.local/share/fonts"
+[ ! -d "${USER_FONT_DIR}" ] && mkdir -pv "${USER_FONT_DIR}"
```

Comment:
Would this make more sense in Lua, in the fairly rare circumstance the user actually taps on the menu item?

Reply:
Right now the menu item is disabled if the folder don't exist (which could happen only on the emulator). Your idea seems better: item always enabled and mkdir on demand.